docs/markdown/Writing Plugins/common-plugin-tasks/plugin-upgrade-guide.md

```diff
@@ -9,6 +9,13 @@ updatedAt: "2022-07-25T20:02:17.695Z"
 2.15
 ----
 
+### Subsystems used in `fmt`/`lint`/`check`/etc... should inherit from `GoalToolSubsystem`
+
+The new subsystem base is declared in `pants.options.subsystem`, and provides the `skip` option.
+
+If your subsystem participates in a goal like `fmt` or `check`, declare it as a base class, and remove
+your `skip` option.
```

Comment:
This should probably be clarified to explain you should still subclass what you did before, but now also the mixin. And probably give an example